addon/utils/polyfills.js

```diff
@@ -0,0 +1,5 @@
+// https://github.com/adopted-ember-addons/ember-data-factory-guy/issues/479
+// Delete this polyfill when ready to drop support for IE11
```

Comment:
We've already dropped support for IE11.

Reply:
The BU app has dropped support for IE, but ember-data-factory-guy hasn't. I opened [this same PR upstream](https://github.com/adopted-ember-addons/ember-data-factory-guy/pull/480) in that repo, and it's more likely to get merged in if it is backwards-compatible.